src/main/webapp/add_event_page/add_event.html

```diff
@@ -7,40 +7,43 @@
     <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDlz0NQ1mc6_UU6OHExGKIrpnkJ5q9BqPs"></script>
     <script src="script.js"></script>
   </head>
-  <body onload="initMap()">
+  <body>
     <div id="header">
       <h1>Add New Event</h1>
     </div>
 
     <div id="form">
-        <form action="/new_event.php">
+        <form>
             <label for="name">Event Name:</label>
             <input type="text" id="name" name="name"><br>
-            <label for="desc">Description:</label>
-            <textarea id="desc" name="desc"></textarea><br>
-            <label for="loc">Building Name:</label>
-            <input type="text" id="loc" name="loc"><br>
-            <label for="date">Date:</label>
-            <input type="text" id="date" name="date"><br>
+            <label for="description">Description:</label>
+            <textarea id="description" name="description"></textarea><br>
+            <label for="location_name">Building Name:</label>
+            <input type="text" id="location_name" name="location_name"><br>
+            <label for="date">Date and Time:</label>
+            <input type="datetime-local" id="date" name="date"><br>
             <label>Event Type:</label>
-            <select name="ev_type" id="ev_type">
-              <option value="Social">Study</option>
+            <select name="event_type" id="event_type">
+              <option value="Study">Study</option>
               <option value="Social">Social</option>
             </select><br>
-            <label>Subject:</label>
+            <!--To be changed later-->
+            <label>Subject:</label> 
```

Comment:
When `Social` is selected, we should hide the subjects. This can be done with an onchange function and changing the style to `display: none`.